.github/workflows/bundle_with_dakota_linux.yml

```diff
@@ -71,6 +71,12 @@ jobs:
         key: carolina_dist_${{ matrix.os }}_python-${{ matrix.python-version }}_boost-${{ inputs.BOOST_VERSION }}_dakota-${{ inputs.DAKOTA_VERSION }}_seba-${{ inputs.SEBA_TAG }}
         path: ./carolina_dist
 
+    - name: Checkout Carolina
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+        path: carolina-main
```

Comment:
Checking out in root folder seems to cause flushing of that folder.
Best use another folder when checking out.